### PR TITLE
Add hue scene transition variable

### PR DESCRIPTION
**Pull request by Nixon506E**
## Proposed change

Add documentation for using the transition variable in hue.hue_activate_scene

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
 https://github.com/home-assistant/core/pulls#issue-564682238
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/hue.markdown [hunk 1]
```diff
@@ -80,12 +80,14 @@ script:
...
           group_name: "Porch"
           scene_name: "Porch Orange"
+          transition: 30
 ```
 
 | Service data attribute | Optional | Description                                                           |
 | ---------------------- | -------- | --------------------------------------------------------------------- |
 | `group_name`           | no       | The group/room name of the lights. Find this in the official Hue app. |
 | `scene_name`           | no       | The name of the scene. Find this in the official Hue app.             |
+| 'transition'           | yes      | The time in 100s of milliseconds to transition to the scene.          |
```
Comment (balloob) [inline source/_integrations/hue.markdown]: what does 100s of milliseconds mean?

File: source/_integrations/hue.markdown [hunk 2]
```diff
@@ -80,12 +80,14 @@ script:
         data:
           group_name: "Porch"
           scene_name: "Porch Orange"
+          transition: 30
```
Comment (balloob) [inline source/_integrations/hue.markdown]: keep examples as simple as possible, leaving optional fields out
```suggestion
```

File: source/_integrations/hue.markdown (hunk 1 above)
Comment (Nixon506E) [reply source/_integrations/hue.markdown]: That is how the Hue api is by default, i.e. a value of 4 is 400 milliseconds.

Comment (balloob) [reply source/_integrations/hue.markdown]: Ah! Now I get it. Let's add an example to clarify it for other readers:

```suggestion
| 'transition'           | yes      | The time in 100s of milliseconds to transition to the scene. For example, a value of 4 means 400 milliseconds.          |
```

**Review by balloob — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**